cmd/soroban-rpc/internal/ingest/service.go

```diff
@@ -251,6 +251,7 @@ func (s *Service) ingest(ctx context.Context, sequence uint32) error {
 	if err := tx.Commit(sequence); err != nil {
 		return err
 	}
+	s.logger.Debugf("Ingested ledger %d", sequence)
```

Comment:
maybe a nitpick and not the time/place for this comment, but still noting here:

would it be better to have something like:

```go
var err error
defer func() {
  s.logger
    .WithError(err)
    .WithField("took", time.Since(startTime))
    .Debugf("Done ingesting ledger %d", sequence)
}()

ledgerCloseMeta, err = s.ledgerBackend.GetLedger(ctx, sequence)
// etc.
```
where each subsequent line assigns that `err` value?

Reply:
The error is surfaced back as a return value, which will be printed later on as a long line so I don't think we should be printing it

The debug printout is to debug possible concurrency problems, which I think does the job